src/main/java/lex/parser/command/Command.java

```diff
@@ -0,0 +1,15 @@
+package lex.parser.command;
+
+/**
+ * Represents a command.
+ */
+public interface Command {
```

Comment:
I like that you created a separate Command Class just to handle all the different commands. An OOP implementation is always more scalable and better designed.